tiledb/rest/capnp/query.cc

```diff
@@ -84,62 +77,64 @@ tiledb::sm::Status query_serialize(
         kj::Array<::capnp::word> protomessage = messageToFlatArray(message);
         kj::ArrayPtr<const char> message_chars = protomessage.asChars();
 
-        const auto nbytes = message_chars.size() + totalNumOfBytesInBuffers +
-                            totalNumOfBytesInOffsets +
-                            totalNumOfBytesInVarBuffers;
-        RETURN_NOT_OK(serialized_buffer->realloc(nbytes));
+        // Write the buffer headers
```

Comment:
Isn't this writing the entire query object not just the buffer headers?

Reply:
Yes, you are right -- I'll update the comment.